foundation_cms/legacy_apps/donate_banner/factory.py

```diff
@@ -12,5 +12,6 @@ class Meta:
 
     name = Faker("sentence", nb_words=3)
     title = Faker("sentence", nb_words=6)
+    foreground_image = SubFactory(ImageFactory)
```

Comment:
Updated because Donate Banner tests were failing due to no foreground image. This is now a required field